src/ser/mod.rs

```diff
@@ -94,6 +96,26 @@ impl Default for PrettyConfig {
             new_line: "\r\n".to_string(),
             indentor: "    ".to_string(),
             separate_tuple_members: false,
+            struct_names: false,
+            _phantom: PhantomData,
+        }
+    }
+}
+
+impl PrettyConfig {
+    #[allow(dead_code)]
```

Comment:
Why allow dead code and why is this function private?

Reply:
my bad. made it public, removed attribute